src/extract/ast-extractors/extract-typescript-deps.js

```diff
@@ -86,25 +86,39 @@ function isRequireCallExpression(pASTNode) {
         typescript.SyntaxKind[pASTNode.expression.originalKeywordKind] === 'RequireKeyword';
 }
 
+function isDynamicImportExpression(pASTNode) {
```

Comment:
There are also import types in case you want to handle them as well:

```ts
var v: import('./types').T;
```

They can also occur in JSDoc of JS files:

```js
/** @type {import('./types').T} */
var v;
```

You might be interested in my library `tsutils`. It has a function `findImports` that handles imports, import equals, re-exports, import expressions, import types and in JS files it also finds require and import types in JSDoc.

Reply:
Thanks! And idd - I'm using these type imports in jsdoc. I'll checkout tsutils. The less custom code the better.

(I also realised that I probably need to handle AMD as well to be complete :-/)

Reply:
I've taken a peek at tsutils - I'm going to use it, although in a separate PR (and import types, and JSDoc imports will get supported then automatically.